tests/test_integration_layout.py

```diff
@@ -19,13 +19,12 @@
 SCREEN_SIZE = Size(SCREEN_W, SCREEN_H)
 PLACEHOLDERS_DEFAULT_H = 3  # the default height for our Placeholder widgets
 
-# TODO: Brittle test
```

Comment:
Not really brittle :smile: : these integration tests have one job, which is to spot regressions. 
They just did their duty when a PR recently brought a breaking change (new scrollbar default size) :slightly_smiling_face: